src/libraries/System.Text.Encodings.Web/src/System/Text/Encodings/Web/DefaultJavaScriptEncoderBasicLatin.cs

```diff
@@ -105,22 +109,18 @@ public override unsafe int FindFirstCharacterToEncode(char* text, int textLength
 
                 if (NeedsEscaping(*(char*)ptr))
                 {
-                    goto Return;
+                    return idx;
```

Comment:
> return idx; [](start = 19, length = 12)

Just double check that you are not making C# unhappy by changing any of the `goto` to `return`.
Reference: [https://source.dot.net/#System.Private.CoreLib/SpanHelpers.Byte.cs,1470](https://source.dot.net/#System.Private.CoreLib/SpanHelpers.Byte.cs,1470)

Reply:
Interesting, thanks for sharing.